### PR TITLE
Feature/swagger fix

### PR DESCRIPTION
**Pull request by garyluu**
This fixes the issue with swagger quay code gen

### CODE REVIEW

File: swagger-java-quay-client/pom.xml [hunk 1]
```diff
@@ -33,14 +34,19 @@
             <plugin>
                 <groupId>io.swagger</groupId>
                 <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>2.2.2</version>
```
Comment (denis-yuen) [inline swagger-java-quay-client/pom.xml]: There's something strange going on here. 
This generates the swagger classes as a client for talking to quay.io

File: dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java [hunk 2]
```diff
@@ -33,6 +34,7 @@
  *
  * @author dyuen
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java]: On the other hand, this is not for talking to quay.io. 
This is our own web service. 

Why is this being modified when the issue is communication with quay.io?
What is the unknown property that needs to be ignored when serializing and deserializing WorkflowVersions?

File: swagger-java-quay-client/pom.xml (hunk 1 above)
Comment (garyluu) [inline swagger-java-quay-client/pom.xml]: Yes, this generates the swagger classes

File: dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java (hunk 2 above)
Comment (garyluu) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java]: When editing a tool tag, workingDirectory is unrecognized when deserializing Tag.
When editing a workflow tag, workingDirectory is unrecognized when deserializing WorkflowVersions.
There are many others.  Not sure what's changed.

Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java]: Hi,
Please investigate what changed. Is workingDirectory a recent addition? Maybe the swagger library is changing without us noticing (is the version pinned?) As it stands, this seems really odd.

**Comment by denis-yuen** [comment]
To be clear, I'm a bit less concerned about the specific new annotations but more concerned about what is actually changing because these two components should be unrelated unless there is a third change outside the scope of this pull request. 
I'm more concerned about unexpected side-effects due to this unknown change.

**Comment by denis-yuen** [comment]
@garyluu  FYI, just to eliminate a red herring for you. 
https://bitbucket.org/api/1.0/repositories/DockstoreTestUser/dockstore-whalesay/raw/master/Dockstore.wdl doesn't exist and doesn't seem to ever have
https://bitbucket.org/api/1.0/repositories/DockstoreTestUser/dockstore-whalesay/raw/master/Dockstore.cwl does

The web service warns about the former not existing, but this is just misleading (and expected).

**Comment by garyluu** [comment]
Rebased, found a magic flag that prevents the new Jackson provider from interfering with our web service Jackson provider.

**Pull request merged**